modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -92,6 +92,135 @@ using the `Flex` component and to replace the `spacing` prop with the `gap` prop
 > **Note**: If you are consuming `StackProps` for a custom component, please make sure to change the
 > `spacing` prop to the `gap` prop
 
+---
+
+### Menu in preview-react
+
+We've removed the `Menu` component that was located in `preview-react` and now will be replaced with
```

Comment:
I think we reference thins in preview as `Preview` or `@workday/canvas-kit-preview-react`

Reply:
Updated